src/Data/Arrays.php

```diff
@@ -478,4 +478,42 @@ public static function arrayChangeCaseValue($array, $case = CASE_LOWER)
 
         return $array;
     }
+
+    /**
+     * Remove duplicate values from array.
+     *
+     * @param array      $array The array to work on.
+     * @param string|int $key   Key that need to evaulate.
+     *
+     * @since 3.0.0
+     *
+     * @return array
+     */
+    public static function removeDuplicates(array $array,$key = '')
```

Comment:
I wonder why the specific `$key` has the default value.

I think the default value should be removed because the specific `$key` will be the required argument, not optional.

Reply:
Here key is optional arg, because key only required if working on Multidimensional arrays. in sequential or assoc array key is not required so i make it optional

Reply:
@Lablnet, ok. It makes sense, but I just note that the key will be `index` or `key` name.

The key is required for multidimensional or order array.

Reply:
key will be `=ley` not index, as you may see code i use default php function for handling indexes and assoc arrays